.storybook/assets/base.css

```diff
@@ -33,3 +33,15 @@ body {
 svg:has(symbol):not(:has(use)) {
 	display: none;
 }
+
+/* set back to default h2 styles */
+#variants {
+	font-size: 24px;
+	margin-bottom: 8px;
+	color: rgb(46, 52, 56);
+	font-weight: bold;
+	line-height: normal;
+	letter-spacing: normal;
+	text-transform: none;
+	border-bottom: 1px solid hsla(203, 50%, 30%, 0.15)
+}
```

Comment:
After weighing a few options, this is what I came up with so that the headings could be more consistent with MDX pages. I'm not crazy about it and I'm definitely open to other ideas if anyone's got one!